app/helpers/timeline_helper.rb

```diff
@@ -67,4 +67,8 @@ def public_timeline?
     @public_timeline ||= false
   end
 
+  def status(reservation)
```

Comment:
I'm a little wary of using this generic of a method name in a helper. If another helper implements the same method, I could see them clobbering each other.